### ISSUE TITLE
MonoGame 3.8 TiledMapRenderer MatrixChainEffect needs to be rebuilded

### ISSUE THREAD

**Issue by janfokke** [craftworkgames/MonoGame.Extended#670]
The MatrixChainEffect is for an older release of MonoGame and needs to be rebuilt.

**Comment by janfokke** [comment]
Fix
Install new effect builder: `dotnet tool install -g dotnet-mgfxc`
https://github.com/MonoGame/MonoGame/blob/2a83a3b94c131d962bacb9833080b2f5b03c2b40/Documentation/articles/tools/2mgfx.md


Open `MonoGame.Extended\Source\MonoGame.Extended.Graphics\Effects\Resources\RebuildEffects.bat` and replace `SET TWOMGFX="C:\Program Files (x86)\MSBuild\MonoGame\v3.0\Tools\2mgfx.exe"` with `SET TWOMGFX="mgfxc"`

Run `RebuildEffects.bat`

**Comment by lithiumtoast** [comment]
Nice, thanks for the assist. I'll check this out soon.

**Comment by lithiumtoast** [comment]
I did this but now running any of the samples has the following problem:

```
System.Exception: 'This MGFX effect seems to be for a newer release of MonoGame.'
```

I will need to update to MG 3.8

**Comment by lithiumtoast** [comment]
@janfokke Can verify the problem is resolved in #673?
